### PR TITLE
vita:0.1.0

### PR DESCRIPTION
**Pull request by cadojo**
I am submitting
- [X] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name in conformance with the guidelines
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] ensured that my submission does not infringe upon the rights of a third party
- [X] tested my package locally on my system and it worked
- [X] named this PR as `name:version` of the submitted package
- [X] agree that my package will not be removed without good reason

Description: A modern, themed résumé template. Separate style and content by adding content through the following functions: `#experience`, `#degree`, `#skill`, `#project`, `#social`.

### CODE REVIEW

File: packages/preview/vita/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[package]
+name = "vita"
+version = "0.1.0"
+entrypoint = "vita.typ"
+authors = ["Joey Carpinelli"]
+license = "MIT"
+description = "Modern résumé templates that separate content and style."
+repository = "https://github.com/cadojo/modern-resumes"
```
Comment (JKRhb) [inline packages/preview/vita/0.1.0/typst.toml:8]: I think this needs to vita here as well, right? :)

```suggestion
repository = "https://github.com/cadojo/vita"
```

Comment (cadojo) [reply packages/preview/vita/0.1.0/typst.toml:8]: Sure does! Thanks :)

**Comment by slashformotion** [comment]
I believe template support is to be worked on. See #36

**Comment by cadojo** [comment]
> I believe template support is to be worked on. See #36 

Great! Is the PR good for merge as-is, or is the template file not ideal right now?

**Comment by slashformotion** [comment]
I guess  the typst team want to take some time to think about how templates should be packaged

**Comment by laurmaedje** [comment]
We want to make the experience of using template packages as straight-forward as possible, with the possibility of providing a project template like your `template.typ` and selecting among templates in the the template gallery. So, I would hold off on merging this like with #36.

**Comment by cadojo** [comment]
> We want to make the experience of using template packages as straight-forward as possible, with the possibility of providing a project template like your `template.typ` and selecting among templates in the the template gallery. So, I would hold off on merging this like with #36.

Good deal, I have [three more](https://github.com/cadojo/correspondence) ready to go when the infrastructure is ready!

**Comment by laurmaedje** [comment]
We now accept templates into the repository. The updated README details what changes need to be done for a package to become a template. You can try your template package locally with the new `typst init` command by compiling the latest Typst compiler from the main branch. Alternatively, you can wait a bit until the next release drops. If there are any questions or something is unclear with the new process, please point this out. Thank you for your patience!

**Comment by laurmaedje** [comment]
I'll close this due to inactivity. Feel free to reopen if you want to continue with it!

**Pull request closed without merging**